### ISSUE TITLE
Damage Logs Web removed any plans to re add it?

### ISSUE THREAD

**Issue by ZachPL** [Tommy228/TTTDamagelogs#297]
The web portion of the damagelogs was removed due to the database columns changing is there any plans to re add support for it?

**Comment by MinIsMin** [comment]
There are plans to readd the web code but it takes some time.

**Comment by iagrib** [comment]
Agreed, it's super useful. The fact that web part for 3.0 is not available yet is one of the reasons I'm still sitting at 2.4
